packages/rollup-contracts/contracts/SequencerBatchSubmitter.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* Internal Imports */
+import {DataTypes as dt} from "./DataTypes.sol";
+import {RollupMerkleUtils} from "./RollupMerkleUtils.sol";
+import {CanonicalTransactionChain} from "./CanonicalTransactionChain.sol";
+import {StateCommitmentChain} from "./StateCommitmentChain.sol";
+
+contract SequencerBatchSubmitter {
+  CanonicalTransactionChain canonicalTransactionChain;
+  StateCommitmentChain stateCommitmentChain;
+  address public sequencer;
+
+  constructor(address _sequencer) public {
+    sequencer = _sequencer;
+  }
+
+  function initialize(
+    address _canonicalTransactionChain,
+    address _stateCommitmentChain
+  ) public onlySequencer {
+    canonicalTransactionChain = CanonicalTransactionChain(_canonicalTransactionChain);
+    stateCommitmentChain = StateCommitmentChain(_stateCommitmentChain);
+  }
+
+  function appendTransitionBatch(
```

Comment:
Could you add a bit about what the bytes of the `_txBatch` and `_stateBatch` represent? Is it just tx calldata? Is it signed transactions, how are they encoded, etc.?